### DISCUSSION TITLE
Dynamic style change

### DISCUSSION THREAD

**Discussion by Kovalskiii** [rnmapbox/maps#2926] [Q&A]
I need to dynamically change style of SymbolLayer (only for pressed cluster). 
I try to do something like this: 
`                    iconImage: [
                      'case',
                      ['==', ['get', 'cluster_id'], +pressedClusterId],
                      ['call', ['myFunc', ['get', 'cluster']]],
                      'defaultMarkerIcon',
                    ]`
                    Is it possible to call here function which will return string with icon name ?
Or there exist more simple way to do this ?
Thank you in advance!

**Comment by uzun0ff** [comment]
Have you managed to do it?
